prelude/align.md

```diff
@@ -0,0 +1,47 @@
+This is basically a copy of the `Align` type in the Rust compiler.
+See [Align](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_target/abi/struct.Align.html).
+
+`Align` is always a power of two.
+
+```rust
+pub struct Align { pow2: BigInt }
```

Comment:
I don't think we want to use the `pow2` representation that the Rust compiler uses. This will be just unnecessarily painful in Coq.